.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - git clone https://github.com/OCA/account-financial-tools $HOME/account-financial-tools -b ${VERSION}
```

Comment:
Needed for the dependency on `account_credit_control_dunning_fees`. I guess the `oca_dependencies.txt` file was not existing at the time of creation of this PR. I'll change that.